Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -369,6 +373,30 @@ public override string ToString()
         }
     }
 
+    public class Desk
+    {
+        public int width;
+        public int Depth { get; init; }
+        public int Height { get; init; }
+
+        [IonPropertyGetter("length")]
```

Comment:
Why length?

Reply:
I decided to avoid the naming conflict issue here (and let the other test take care of that).
Updated this to "desk width" to avoid confusion.